contracts/contracts/PoolDeposits.sol

```diff
@@ -42,7 +42,12 @@ contract PoolDeposits {
   event DepositWithdrawn(address indexed user);
   event PartialDepositWithdrawn(address indexed user, uint256 amount);
   event ProposalWithdrawn(address indexed benefactor);
-  event InterestSent(address indexed user, uint256 amount, uint256 iteration);
+  event InterestSent(address indexed user, uint256 amount);
+  event WinnerPayout(
```

Comment:
It didn't make sense to me that the payments to the others used the same event as the payout to the winner.

Reply:
Hmmm okay, this was handled fine in the graph and quite obvious there, but understand you want to make it more explicit. Will need to redo some graph code